src/samplers/ext4/stat.rs

```diff
@@ -45,6 +48,83 @@ impl Ext4Statistic {
             Self::FsyncLatency => Some("fsync"),
         }
     }
+
+    #[cfg(feature = "bpf")]
+    pub fn bpf_probes_required(self) -> Vec<FunctionProbe> {
+        // define the unique probes below.
+        let generic_file_read_probe = FunctionProbe {
+            name: String::from("generic_file_read_iter"),
```

Comment:
I feel like it's a bit more idiomatic to use `"...".to_string()`

Reply:
got it, TIL.
Updated all locations